src/ride/ride_structs.jl

```diff
@@ -16,9 +16,9 @@
     save_interim_results::Bool = false
 end
 
-abstract type RideModus end
-struct RideOriginal <: RideModus end
-struct RideUnfold <: RideModus end
+abstract type ModusRide end
```

Comment:
```suggestion
abstract type AbstractRide end
```
As Bene suggested I would rename this